Source/Details/ASCollectionViewLayoutInspector.m

```diff
@@ -27,6 +27,11 @@
 // of the collection view
 ASSizeRange NodeConstrainedSizeForScrollDirection(ASCollectionView *collectionView) {
   CGSize maxSize = collectionView.bounds.size;
+  UIEdgeInsets contentInset = collectionView.contentInset;
+  if (! UIEdgeInsetsEqualToEdgeInsets(UIEdgeInsetsZero, contentInset)) {
+    maxSize.width -= (contentInset.left + contentInset.right);
+    maxSize.height -= (contentInset.top + contentInset.bottom);
```

Comment:
Same principle as in the pager: I think we should not apply these in the scrolling direction.